tools/src/test/java/org/apache/pdfbox/tools/TestAddText.java

```diff
@@ -0,0 +1,64 @@
+package org.apache.pdfbox.tools;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.StringReader;
+
+import junit.framework.TestCase;
+import junit.framework.TestSuite;
+
+import org.apache.pdfbox.pdmodel.PDDocument;
+import org.apache.pdfbox.pdmodel.PDPageContentStream;
+import org.apache.pdfbox.pdmodel.graphics.image.PDImageXObject;
+import org.apache.pdfbox.text.PDFTextStripper;
+import org.apache.pdfbox.tools.AddImage;
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+
+
+public class TestAddText extends TestCase
+{
+    
+    @Test
+    public void testLoadFileAndInitializeStream() throws IOException
+    {
+        AddText add = new AddText();
+        
+        String expected = "../tools/src/main/java/org/apache/pdfbox/tools/AddImageResources/examplePDFWithText.pdf";
+
+        PDDocument actual = add.loadFileandInitializeStream(expected);
+        int actualPages = actual.getNumberOfPages();
+        assertEquals(1, actualPages);
+    }
+
+    @Test
+    public void testWriteText()
```

Comment:
This test fails when I run it on my machine. 

junit.framework.AssertionFailedError: 
Expected :1
Actual   :0

Not sure what you're trying to assert at the end of this test. The actual you entered is 0 which will never pass because 0 never equals 1